### PR TITLE
PAYARA-2171 Add Timeout Parameter to Start Instance Command (Payara-5 maintenance)

### PR DESCRIPTION
**Pull request by MattGill98**
Maintenance PR for #2299.

### CODE REVIEW

**Comment by MattGill98** [comment]
jenkins quick test payara 5 please

**Comment by MattGill98** [comment]
How's that ;)

**Comment by MattGill98** [comment]
jenkins quick test payara 5 please

File: nucleus/cluster/admin/src/main/java/com/sun/enterprise/v3/admin/cluster/StartInstanceCommand.java [hunk 1]
```diff
@@ -297,8 +296,13 @@ private void startInstance(AdminCommandContext ctx) {
 
     }
 
-    // return null means A-OK
-    private String pollForLife(final Server instance) {
+    /**
+     * Poll for the specified amount of time to check if the instance is running.
+     * Returns whether the instance was started after the timeout.
```
Comment (Pandrex247) [inline nucleus/cluster/admin/src/main/java/com/sun/enterprise/v3/admin/cluster/StartInstanceCommand.java]: Surely before the timeout?

Comment (MattGill98) [reply nucleus/cluster/admin/src/main/java/com/sun/enterprise/v3/admin/cluster/StartInstanceCommand.java]: Fair point

**Comment by MattGill98** [comment]
jenkins quick test payara 5 please

**Pull request merged**